APIClient/WeatherForecastApi.cs

```diff
@@ -11,13 +11,17 @@ namespace APIClient
     internal class WeatherForecastApi
     {
         private string uri;
+        const string obsoleteInfo = "use Get method instead";
```

Comment:
This is not necessary. Because is not somethings that should vary. An error is an error, not something that changes over time